edx_proctoring/api.py

```diff
@@ -313,6 +313,12 @@ def get_exam_by_content_id(course_id, content_id):
     """
     proctored_exam = ProctoredExam.get_exam_by_content_id(course_id, content_id)
     if proctored_exam is None:
+        log.exception(
```

Comment:
Please: https://openedx.atlassian.net/wiki/spaces/SUST/pages/125206530/Escalation+Coding+Practices

Reply:
Thanks @awaisdar001. I have change it according to the doc link.